src/node_object_wrap.h

```diff
@@ -30,6 +30,7 @@ namespace node {
 
 class ObjectWrap {
  public:
+  enum InternalFields { kSlot, kInternalFieldCount };
```

Comment:
```suggestion
  enum InternalFields { kSlot, kInternalFieldCount };
  
```
But I'd probably skip the enumification treatment for this file. Node.js itself doesn't use it and, since ObjectWrap is a public API, the enum shows up in classes that inherit from it.